docs/userguides/deactivatedevices.md

```diff
@@ -38,6 +38,9 @@ curating a list of devices to deactivate:
 * `--exclude-most-recently-connected INTEGER` - allows you to exclude the most recently connected device (per user) from the results. This allows you to ensure that every user is left with at least N device(s), regardless of how recently they have connected.
 * `--created-before DATE|TIMESTAMP|SHORT_TIME` - allows you to only see devices created before a particular date.
 
+```eval_rst
+.. note:: If you use Excel to filter or edit the list of devices, you will find that the ``guid`` column is displayed in scientific notation—when you change the column type to text, you'll see that the full number now has had its last digits replaced by zeros. This modification makes the guid invalid for deactivating devices. In order to edit the list in Excel without overwriting this, you will need to `import the data into a new workbook <https://support.microsoft.com/en-us/office/keeping-leading-zeros-and-large-numbers-1bf7b935-36e1-4985-842f-5dfa51f85fe7>`_. rather than just opening the CSV. If you want to work on this in Excel later, you will also want to save as an xlsx file and only 'save as' csv when you're ready to make the final export for running the bulk deactivate command.
```

Comment:
Is the data modified by merely opening the file in Excel or does changing the column type cause the truncation?

Reply:
I just tried this with the `user list` command (And I really am not an Excel user), and what happened for me is that I noticed the `userUid` column was sci-not and when I changed it to text, nothing happened.  It remained in sci-not but the type changed.  When I `cat` when .csv file, the original IDs are still there. Does this mean anything to you?

Reply:
did you save the file anytime after opening it?

Reply:
The data in the underlying csv isn't changed except when saving, but at the same time, if you open the CSV, Excel converts the guids to scientific notation, and if you try to convert that back to text, it has already thrown away the "non-significant" numbers at the end of the guid, so you won't be able to save the CSV with the correct guids. 

In other words, opening the CSV file directly in Excel, rather than importing, already sets the column type to one that throws away numbers in the guid. The csv itself is only modified after you save in Excel, though. 

This is not an uncommon Excel issue even outside of Code42 tooling—searching for "large numbers csv" is mostly users encountering this with Excel. This is a [pretty good writeup of the issue](https://support.3dcart.com/knowledgebase/article/View/619/7/why-does-my-exported-csv-data-get-converted-to-weird-formats) from an e-commerce software provider.